docs/topics/Config-options.md

```diff
@@ -306,7 +307,8 @@ a JSON array of span data.
 Optional common tags to annotate on all submitted Zipkin traces. Can be overwritten by spans at runtime, if keys 
 collide. 
 
-<tip>Recommended common tags: <code>clusterName</code>, <code>env</code></tip>
```

Comment:
Did you intend to remove this?

Reply:
good catch. that was a mistake. restored